src/redux/constants.js

```diff
@@ -0,0 +1,3 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+export const CHECK_BOOK_CATEGORY = 'CHECK_BOOK_CATEGORY';
```

Comment:
- [ ] I'm not able to see your components here since any changes were done in them however, in your `book.js` component, the books rendered on the UI should not be passed from the parent component as props but rather you have to get them from the `store` you just created where you need to be able o add and remove a book through the action.